app/views/catalog/_all_search_results_add_to_cart.html.erb

```diff
@@ -0,0 +1,3 @@
+<div id="add-results-to-cart" class="add-results-to-cart">
+  <%= link_to 'ADD RESULTS TO CART', request.params.merge(action: "add_to_cart", controller: 'all_search_result_ids'), class:"btn btn-outline-secondary" %>
+</div>
```

Comment:
Should we use a ViewComponent instead of a partial?  Or wait, does it NEED to be a partial to work with the Blacklight API you found? Woah. 

We better have really good tests for this stuff, I don't trust Blacklight at all!

Reply:
I believe add_results_collection_tool will only take a symbol that is assumed to be the name of a partial. I did a bit of digging around but was unable to get it to accept a partial.